springboard/tests/test_language.py

```diff
@@ -0,0 +1,82 @@
+from springboard.tests import SpringboardTestCase
+from springboard.views import SpringboardViews
+
+from pyramid import testing
+
+
+class TestLanguages(SpringboardTestCase):
+
+    def setUp(self):
+        self.workspace = self.mk_workspace()
+        settings = {
+            'unicore.repos_dir': self.working_dir,
+            'unicore.content_repo_urls': self.workspace.working_dir,
+            'available_languages': '\n'.join([
+                'eng_GB = English',
```

Comment:
@nathanbegbie you're right, I don't think we need to store this stuff now. @miltontony any specific concerns from your side in removing this?